docs/guides/dns-over-https.md

```diff
@@ -97,39 +97,92 @@ sudo systemctl start cloudflared
 sudo systemctl status cloudflared
 ```
 
-Now test that it is working! Run the following `dig` command, a response should be returned similar to the one below:
+##### Configuring `cloudflared` as a system service
+
+Proceed to create a configuration file for `cloudflared` in `/etc/default/cloudflared`:
 
 ```sh
-dig @127.0.0.1 -p 5053 google.com
+sudo touch /etc/default/cloudflared
+sudo nano /etc/default/cloudflared
+```
+
+Copy the following configuration:
 
+```yml
+proxy-dns: true
+proxy-dns-port: 5053
+proxy-dns-upstream:
+  - https://1.1.1.1/dns-query
+  - https://1.0.0.1/dns-query
+```
+
+Now install the service via `cloudflared`'s [service command](https://developers.cloudflare.com/argo-tunnel/reference/arguments/#service-command):
+
+```sh
+sudo cloudflared service install
```

Comment:
I tried doing this but got an error about the `cert.pem` file. That said, it is likely I did something wrong. In any case, time for a reflash of my Pi. I will try again later (and on a droplet, this time...)

Reply:
I will also try it tomorrow on my Pi, but being I don't currently have it backed up, you know, I'm extra careful :)

Oh, BTW when using this command, there's an extra option being passed which I think we should probably have it in the manual way too. `--no-autoupdate` https://developers.cloudflare.com/argo-tunnel/reference/arguments/#no-autoupdate